processes/runPre.R

```diff
@@ -50,3 +50,12 @@ source('processes/loadLibs.R')
 # compile the c++ file and make available to R
 TMB::compile("assessment/caa.cpp")
 
+
+# need to get the mprocfile from set_om_parameters_global.R
+source('modelParameters/set_om_parameters_global.R')
+
+# Copy set_om_parameters_global.R and mprocfile into the results folder
+
+file.copy(file.path("modelParameters",mprocfile), file.path(ResultDirectory,mprocfile))
```

Comment:
This copies the mproc and set_om_parameters_global.R to the simulation results folder.